docusaurus/docs/reference/config-types/host.v2.md

```diff
@@ -0,0 +1,291 @@
+---
+sidebar_label: host.v2
+sidebar_position: 20
+---
+
+# The `Host.v2` Config Type
+
+The `host.v2` configuration type defines how edge routers or tunnelers should create terminators and
+health checks for the application endpoints that provide a service.
+
+## Terminators
+
+Each config can define one or more terminators. If a service is hosted by multiple servers, the
```

Comment:
I personally think "terminators" is confusing. I know that's the field name, so referencing it as such makes sense to me. The "terminators" array is really an array of `host.v1` entries, right? I'd advocate that we simplify the host.v2 by putting all this nice detail into the `host.v1` structure/doc and making this page be more of a "this structure exists to create/aggregate host.v1 entities".